src/Parser/DeathKnight/Unholy/Modules/Features/RpPoolingDA.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import SpellIcon from 'common/SpellIcon';
+import SpellLink from 'common/SpellLink';
+
+import Analyzer from 'Parser/Core/Analyzer';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+
+
+class RpPoolingDA extends Analyzer {
+  static dependencies = {
+    combatants: Combatants,
+  };
+  on_initialized() {
+    this.active = this.combatants.selected.hasTalent(SPELLS.DARK_ARBITER_TALENT.id);
+  }
+
+  totalRunicPowerPooled = 0;
+  totalDarkArbiterCasts = 0;
+  currentRP = 0;
+  maxRP = 0;
+  
+   on_toPlayer_energize(event){
+	   this.currentRP = event.classResources[0]['amount']/10 || 0;
```

Comment:
Since a player can have multiple resources active at the same time, RP might not always be at the same index. I recommend doing a `event.classResources.find` for the resource type instead. See the `RageWasted` module for an example.

Reply:
The order of this expression can be error sensitive; `undefined / 10` will result in NaN which is one of the most unpredictable types in JS there is so should be avoided at all times. It would be safer to write this as `(event.classResources[0]['amount'] || 0) / 10`.